web/Annotation/annotation-comments/README.md

```diff
@@ -0,0 +1,39 @@
+# Annotation Comments POC
+
+This is a proof-of-concept project showcasing the usage of comment linked to annotations automatically with PSPDFKit. Please note that this project is built using a [nightly build](https://pspdfkit.com/guides/web/troubleshooting/nightlies/) version of PSPDFKit. For production applications, we recommend using the official stable version.
```

Comment:
One question, is this a recently created example that actually requires our nightly still?

Reply:
I have created this POC this month and I don't think we have released new version after that. It utilises the new comments UI using BaselineUI